gilbreth_support/include/gilbreth_support/urdf_creator.h

```diff
@@ -0,0 +1,158 @@
+#ifndef GILBRETH_SUPPORT_URDF_CREATOR_H
+#define GILBRETH_SUPPORT_URDF_CREATOR_H
+
+#include <tinyxml.h>
+
+namespace gilbreth
+{
+namespace simulation
+{
+
+struct InertiaTag
+{
+  InertiaTag(const std::string& ixx = "1",
```

Comment:
I personally prefer to avoid  using reference variables on arguments that get default values, but this is a legal statement nonetheless, see [here ](https://stackoverflow.com/questions/14612933/const-reference-default-value)for more on the effects of initializing a const reference to a temporary object if you're interested.